x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/rollup/v2/RollupActionSingleNodeTests.java

```diff
@@ -305,36 +364,93 @@ private void bulkIndex(SourceSupplier sourceSupplier) throws IOException {
         if (bulkResponse.hasFailures()) {
             fail("Failed to index data: " + bulkResponse.buildFailureMessage());
         }
-        assertHitCount(client().prepareSearch(index).setSize(0).get(), docCount);
+        assertHitCount(client().prepareSearch(indexName).setSize(0).get(), docCount);
     }
 
     private void rollup(RollupActionConfig config) {
+        rollup(index, rollupIndex, config);
+    }
+
+    private void rollup(String sourceIndex, String rollupIndex, RollupActionConfig config) {
         AcknowledgedResponse rollupResponse = client().execute(RollupAction.INSTANCE,
-            new RollupAction.Request(index, rollupIndex, config)).actionGet();
+            new RollupAction.Request(sourceIndex, rollupIndex, config)).actionGet();
         assertTrue(rollupResponse.isAcknowledged());
     }
 
+    private RolloverResponse rollover(String dataStreamName) throws ExecutionException, InterruptedException {
+        RolloverResponse response = client().admin().indices().rolloverIndex(new RolloverRequest(dataStreamName, null)).get();
+        assertTrue(response.isAcknowledged());
+        return response;
+    }
+
     private void assertRollupIndex(RollupActionConfig config) {
```

Comment:
maybe this method should be removed and replaced with direct calls to `assertRollupIndex(config, index, rollupIndex)` so things are more explicit

Reply:
Thanks a lot for the feedback and all the help.

I am removing `assertRollupIndex(RollupActionConfig config)` and also `rollup(RollupActionConfig config)`